services/web/server/src/simcore_service_webserver/login/_sql.py

```diff
@@ -13,16 +26,23 @@ def find_one_sql(table, filter_, fields=None):
     keys, values = _split_dict(filter_)
     fields = ", ".join(fields) if fields else "*"
     where = _pairs(keys)
-    sql = f"SELECT {fields} FROM {table} WHERE {where}"  # nosec
+    sql = f"SELECT {fields} FROM {table} WHERE {where}"  # noqa: S608 # @pcrespov SQL injection issue here
```

Comment:
@sanderegg add reference to https://github.com/ITISFoundation/osparc-simcore/issues/3652
I will promote this issue